### PR TITLE
SC-14724 Update Sematext Monitoring docs

### PR DESCRIPTION
**Pull request by cpipilas**
Updates on Sematext Monitoring docs.

### CODE REVIEW

File: docs/monitoring/index.md [hunk 1]
```diff
@@ -36,12 +36,13 @@ Follow along the [Quick Start](./quick-start) guide to learn how.
...
 App settings and actions include, but are not limited to:
 
-  - inviting others to your App
-  - [alert rules](../alerts)
   - changing your App's plan
-  - connecting Apps
-  - heartbeat alert cration
-  - App ownership transfer
+  - [managing alert rules](../alerts)
+  - [connecting Apps](https://sematext.com/docs/guide/integrations-guide/#connected-apps)
```
Comment (otisg) [inline docs/monitoring/index.md:41]: @cpipilas Note there is a ticket in next sprint to add a page for Connected Apps.  We'll need to update any links to this section, like this one, when we do this.

File: docs/monitoring/index.md [hunk 2]
```diff
@@ -36,12 +36,13 @@ Follow along the [Quick Start](./quick-start) guide to learn how.
...
+  - [heartbeat alert creation](https://sematext.com/docs/alerts/creating-heartbeat-alerts/)
+  - [inviting others to your App](https://sematext.com/docs/team/app-guests/)
+  - [App ownership transfer](https://sematext.com/docs/team/transfer-apps/)
+  - [setting up a Sematext Monitoring Agent to ship metrics](https://sematext.com/docs/monitoring/quick-start/#setting-up-monitoring-agents)
   - scheduled report emails (aka Subscriptions)
```
Comment (otisg) [inline docs/monitoring/index.md:46]: @cpipilas I think there is a ticket for this getting a page, too.

Comment (cpipilas) [reply docs/monitoring/index.md:44]: Added subtask in the relevant ticket -> https://sematext.atlassian.net/browse/SC-14812

**Review by fulyauluturk — COMMENTED** [review]
LGTM

**Pull request merged**